pyrep/objects/octree.py

```diff
@@ -0,0 +1,107 @@
+from pyrep.objects.object import Object
+from pyrep.const import ObjectType
+from pyrep.backend import sim
+
+class Octree(Object):
+    """An octree object."""
+
+    @staticmethod
+    def create(voxelSize, pointSize=None, options=0) -> 'Octree':
```

Comment:
Could we add types to the input arguments e.g. `voxel_size: int`.
Also, for the front end, could you use snake_case rather than CamelCase for the front end variables?